src/components/TableResize.js

```diff
@@ -94,21 +94,32 @@ class TableResize extends React.Component {
   };
 
   onResizeStart = (id, e) => {
-    this.setState({ isResize: true, id, startPosition: e.clientX });
+    this.setState({ isResize: true, id });
   };
 
   onResizeMove = (id, e) => {
-    const { startPosition, isResize, resizeCoords } = this.state;
-
+    const { isResize, resizeCoords } = this.state;
+    const fixedMinWidth = 100;
+    const idNumber = parseInt(id);
+    const finalCells = Object.entries(this.cellsRef);
+    if (idNumber >= finalCells.length) return;
     if (isResize) {
-      const leftPos = startPosition - (startPosition - e.clientX);
+      let leftPos = e.clientX;
+      if (leftPos > resizeCoords[idNumber + 1].left - fixedMinWidth)
```

Comment:
Looking for `resizeCoords` at an array position larger than the id sometimes causes it to be undefined. Can you explain what situations would cause us to have a `resizeCoords` array that is a larger index than the number of columns we have?

Reply:
changed `if (idNumber >= finalCells.length)` to `if (idNumber >= finalCells.length-1)` - as this line of code was added to prevent error with undefined id when trying to move last resizer. Last resizer should not be moved at all.

Reply:
Thanks for explaining. Your catch here raises an interesting question about the last resizer. We really don't want users even attempting to resize on the edge of the table, so I think it would be better to not render a resizer for the last column at all. That way, we don't invite confusion with resize cursors and we can simplify the logic here a bit since we know we can't possibly resize on the last column.

Reply:
Problem is, that with present logic, movement of second last resizer is limited by position of last one. If we will remove last one - will need to rebuild code for 2nd last to be limited by total width of table. If you want to test removing of last resizer - this can be easily done by adding `finalCells.pop();` after `finalCells `declaration in `setDividers = () => {`

Reply:
Removed last resizer - it complicated a bit checks as needed to eliminate case when last resizer is the only one, so it would be first as well. Hope this split to functions makes code bit more readable.

Reply:
Another way to do this would be to keep `resizeCoords` as is, and create a new array from it that has the final element removed, and use that array to draw the coordinates. The advantage there would be that we would not have to rethink the logic that uses `resizeCoords`. However, what you have also works!

Reply:
I already rewrote this logic, plus did not see a point to store this last resizer only for that purpose.
Instead to position of last resizer, prefered to keep reference to table width.

Reply:
@gabrielliwerant - You wait something from my end, or you are still testing and I should wait?